_posts/2021-05-23-irc-move.md

```diff
@@ -0,0 +1,41 @@
+<!-- more -->
+
+## What is IRC?
+
+In order to provide a bit of a background, what follows will be short explanation what all this is. If you are familiar with IRC, skip to the next section.
+
+[freenode](https://freenode.net/) and [Libera.Chat](https://libera.chat) are IRC networks. IRC stands for Internet Relay Chat. Without diving too deep in technical explanations (if you care for this, read the [wikipedia article](https://en.wikipedia.org/wiki/Internet_Relay_Chat)), it is an instant communication standard for groups. You install an IRC app on your device (or use a webapp), those act as IRC clients, which you then need to point to an IRC server. Bigger IRC servers are called networks, because instead of just one server, they are a network of several servers (makes sense). The biggest and oldest IRC network is called freenode, and we as NewPipe used this network to host our channel. Up until this week.
+
+## What has happened?
+
+This will be a very short explanation again. This time, we will condense what apparently has happened behind the scenes of freenode. If you want to paint your own picture, read [this post](https://gist.github.com/realrasengan/88549ec34ee32d01629354e4075d2d48) from one side of this drama, and [this resigning letter](https://gist.github.com/aaronmdjones/1a9a93ded5b7d162c3f58bdd66b8f491) from the other. There are many more resigning letters around, you can search for it.
+
+In a nutshell, about a dozen volunteer developers behind freenode left the project. This happened because Andrew Lee bought freenode Limited in 2017, a company which was used as a legal representation of freenode, mostly for their freenode Live Conferences. While he promised to not obtain any influence over freenode itself, this apparently changed this week. Andrew and his team gained full controll over the network (as stated by their [twitter account](https://twitter.com/freenodestaff/status/1395046345145307140)), which lead to the volunteers resign. Whilste resigning, mosted of the volunteers stated they will now be at Libera.Chat.
+
+## Why is NewPipe moving?
+
+Honestly, we do not trust an American business man saying he won't do anything with our data. We do trust developers, we are developers ourself. When you do something to a project which leads to its core team resigning, and said core team starts a new project, we will join this new project. We like our IRC channels, they are the best way to communicate with each other while staying True to our privacy values. After discussing the events internally, we decide to switch the network, and we are in good company doing so. 
```

Comment:
```suggestion
Honestly, we do not trust an American businessman saying he won't do anything with our data. We do trust developers, as we are developers ourselves. When someone does something to a project which leads to its core team resigning, and said core team starts a new project, we prefer to join the new project. We like our IRC channels; they are the best way to communicate with each other while staying true to our privacy values. After discussing the events internally, we decided to switch networks, and we are in good company doing so.
```

Reply:
What about mentioning examples of other major orgs that are switching, like Ubuntu's IRC channels?

Reply:
thought about it, not sure if it was fitting or not.